src/components/field-types/CheckboxFieldComponent.vue

```diff
@@ -17,6 +16,10 @@
           :disabled="field.disabled">
         <label :for="field.id + '-' + index" class="form-check-label">{{ option.label }}</label>
       </div>
+      <p>
+        <button class="select-all btn btn-link btn-sm" @click.prevent="selectAll"><i>Select all</i></button>
```

Comment:
- Is prevent needed here? unless its of type submit, it should not trigger a page load I think
- Why not use <a>?